extension/api.js

```diff
@@ -83,10 +88,20 @@
 			url += 'api/v3/notifications';
 		}
 
-		url += participating;
+		url += `?${query.join('&')}`;
 
 		xhr('GET', url, opts, (data, status, response) => {
```

Comment:
GitHub people say that no real-time stuff is available, so I have to be smarter about handling notification responses ... :disappointed: